Scripts/build.sh

```diff
@@ -1,2 +1,20 @@
-echo "xcodebuild build -workspace $TRAVIS_XCODE_WORKSPACE -scheme $TRAVIS_XCODE_SCHEME -sdk $TRAVIS_XCODE_SDK"
-xctool build test -workspace "$TRAVIS_XCODE_WORKSPACE" -scheme "$TRAVIS_XCODE_SCHEME" -sdk "$TRAVIS_XCODE_SDK"
+#checking if xcpretty is available to use
+command -v xcpretty >/dev/null
+if [ $? -eq 1 ]; then
+echo >&2 "xcpretty not found don't use it."
```

Comment:
I am installing it in travis.yaml.

On Friday, 30 January 2015, Sendhil Panchadsaram <notifications@github.com>
wrote:

> In Scripts/build.sh
> <https://github.com/wordpress-mobile/WordPress-iOS/pull/3151#discussion_r23819103>
> :
>
> > @@ -1,2 +1,20 @@
> > -echo "xcodebuild build -workspace $TRAVIS_XCODE_WORKSPACE -scheme $TRAVIS_XCODE_SCHEME -sdk $TRAVIS_XCODE_SDK"
> > -xctool build test -workspace "$TRAVIS_XCODE_WORKSPACE" -scheme "$TRAVIS_XCODE_SCHEME" -sdk "$TRAVIS_XCODE_SDK"
> > +#checking if xcpretty is available to use
> > +command -v xcpretty >/dev/null
> > +if [ $? -eq 1 ]; then
> > +echo >&2 "xcpretty not found don't use it."
>
> Will the build still happen if xcpretty isn't available?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wordpress-mobile/WordPress-iOS/pull/3151/files#r23819103>
> .
>